plugins/ragdoll/plugin/commands/sh_fall.lua

```diff
@@ -6,7 +6,7 @@ COMMAND.aliases = { 'fallover', 'charfallover' }
 COMMAND.no_console = true
 
 function COMMAND:on_run(player, delay)
-  delay = math.clamp(tonumber(delay) or 0, 2, 60)
+  delay = math.clamp(tonumber(delay) or 0, 0, 60)
```

Comment:
This enables an exploit where people can type `/fall 0.1` and immediately get up. A better solution desired.